backend/de.metas.servicerepair.base/src/main/java/de/metas/servicerepair/project/service/commands/createQuotationFromProjectCommand/QuotationLineAggregator.java

```diff
@@ -122,18 +108,18 @@ public void createOrderLines(@NonNull final OrderFactory orderFactory)
 				.qty(qty)
 				.manualPrice(isZeroPrice() ? Money.zero(priceCalculator.getCurrencyId()) : null)
 				.description(description)
+				.hideWhenPrinting(isHideWhenPrinting())
 				.details(details);
 	}
 
-	public QuotationLineAggregator zeroPrice(@Nullable final Boolean zeroPrice)
+	private boolean isZeroPrice()
 	{
-		this.zeroPrice = zeroPrice;
-		return this;
+		return zeroPrice != null ? zeroPrice : key.isZeroPrice();
 	}
 
-	private boolean isZeroPrice()
+	private boolean isHideWhenPrinting()
 	{
-		return zeroPrice != null ? zeroPrice : key.isZeroPrice();
+		return key.getType() == ServiceRepairProjectCostCollectorType.RepairedProductToReturn;
```

Comment:
@cristinamghita  i think this is not OK. it shall be `SparePartsOwnedByCustomer`.